packages/ember-metal-views/lib/renderer.js

```diff
@@ -63,7 +63,7 @@ function Renderer_renderTree(_view, _parentView, _insertAt) {
     if (!contextualElement && parent && parent._childViewsMorph) {
       contextualElement = parent._childViewsMorph.contextualElement;
     }
-    if (!contextualElement && view._didCreateElementWithoutMorph) {
+    if (!contextualElement) {
```

Comment:
@mixonic Sounds good. I think the solution I landed on (with `renderToElement()`) is a better fix anyway, and doesn't require this change. I've removed it.